src/screens/Home/index.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+import { Text, View, Button } from 'react-native'
+import NavigationService from '../../navigationService'
+import * as firebase from 'firebase'
+
+const Home = () => {
+  const LogOut = async () => {
+    await firebase.auth().signOut()
+    NavigationService.navigate('SignUp')
+  }
```

Comment:
If this function is not accessing anything form its closure, move its definition to the top level. That way it'll be created only once and not every time the Home component renders.